spec/services/pul_metadata_services/pulfa_record/media_resource_attributes_spec.rb

```diff
@@ -1,14 +0,0 @@
-# frozen_string_literal: true
-require "rails_helper"
-
-describe PulMetadataServices::PulfaRecord::MediaResourceAttributes do
-  subject(:media_resource_attributes) { described_class.new(data) }
-  let(:source) { file_fixture("pulfa/C0652/c0377.xml").read }
-  let(:data) { Nokogiri::XML(source).remove_namespaces! }
-
-  describe "#title" do
-    it "extracts the title value for media resource metadata" do
-      expect(media_resource_attributes.title).to eq ["Emir Rodriguez Monegal Papers"]
```

Comment:
Were there other tests ensuring we got the desired title?

Reply:
See commit message for second commit. Basically the logic for this was backwards according to a comment, and it doesn't make sense either - there's no reason to grab the collection title instead of the component title.

Reply:
👍🏻 @tpendragon pointed me at a test that checks for the desired title: https://github.com/pulibrary/figgy/blob/2a145c62ab8847f31a3b576b97d9f63d7a725354/spec/jobs/ingest_archival_media_bag_job_spec.rb#L264-L279